server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/DefaultMirroringServicePlugin.java

```diff
@@ -47,12 +50,30 @@ public synchronized CompletionStage<Void> start(PluginContext context) {
         DefaultMirroringService mirroringService = this.mirroringService;
         if (mirroringService == null) {
             final CentralDogmaConfig cfg = context.config();
+            final Optional<PluginConfig> optional =
+                    cfg.pluginConfigs().stream().filter(plugin -> configType().isInstance(plugin))
```

Comment:
nit; I think it may be worth just constructing a map `Class -> PluginConfig` in `CentralDogmaConfig` so that we don't have to do this every time. I think it's fine not to do this in this PR though.

Reply:
That's a good suggestion. Addressed. 👍 
Note that I've added the additional `pluginConfigMap` method and left the `pluginConfigs()` as is for JSON serialization.